client/src/app/(metadraft)/(validator)/data-validation/assets/grid/card.tsx

```diff
@@ -14,20 +14,33 @@ import {
 } from "~/components/ui/popover";
 
 import { ImageWithFallback } from "~/components/image-with-fallback";
-import Status from "../table/status";
+import Status from "../status";
 import Actions from "../actions";
+import { useRouter } from "next/navigation";
 
 export default function Card({ metadata }: { metadata: MetadataCollection }) {
+  const router = useRouter();
   return (
     <div
       className={cn(
-        "relative flex flex-col rounded-xl border border-white/20 bg-card",
+        "relative flex flex-col rounded-xl border border-white/20 bg-card hover:border-white",
       )}
     >
-      <ImageWithFallback src={getImageSrc(metadata.metadata.image)} />
+      <ImageWithFallback
+        src={getImageSrc(metadata.metadata.image)}
+        className={"cursor-pointer"}
+        onClick={() => router.push(`/metadata/${metadata.id}`)}
+      />
       <div className="flex flex-col gap-4 p-4">
         <Status metadata={metadata} />
-        <Typography as="largeText">{metadata.assetName}</Typography>
+
+        <Typography
+          as="largeText"
+          className="cursor-pointer"
+          onClick={() => router.push(`/metadata/${metadata.id}`)}
```

Comment:
make the whole card a Link component instead of doing this onclick thing on child components

add `<span class="sr-only">go to asset page</span>` inside the link component, as well as a title and an aria-label to the link 'go to asset page'  so that it is labeled